apidoc/Titanium/Network/HTTPClient.yml

```diff
@@ -536,6 +564,26 @@ properties:
         On iOS, the default is 15000 (15 seconds).
     type: Number
 
+  - name: timeoutForResource
+    summary: |
```

Comment:
Link to the apple docs for the underlying property we're surfacing? https://developer.apple.com/documentation/foundation/nsurlsessionconfiguration/1408153-timeoutintervalforresource?language=objc

Reply:
Also does this property only matter if `waitsForConnectivity` is `true`? If so, may want to link to that property in the description here or note it in some way.

Reply:
1. Added apple doc link. 
2. This property is not dependent on `waitsForConnectivity`  is `true`. Vice versa is true. If `waitsForConnectivity` is `true` then it matters. It is documented in `waitsForConnectivity`

Reply:
So, it *does* only matter if `waitsForConnectivity === true`? Please state so in the summary or description.

Reply:
@sgtcoolguy  Sorry for confusion.
1. This property ensures how long to wait for whole resource to transfer before giving up. 
2. If  waitsForConnectivity === true, in that case also this property matters.

Updated doc to explain above.